.buildkite/premerge.steps.yaml

```diff
@@ -7,16 +7,22 @@ agent_transients: &agent_transients
 
 common: &common
   agents:
+    - "agent_count=1"
     - "capable_of_building=gdk-for-unreal"
     - "environment=production"
+    - "machine_type=quad"  # this name matches to SpatialOS node-size names
     - "platform=windows"
     - "permission_set=builder"
-    - "queue=v2-1551960839-2fc9bbe6e15deffd-------z"
     - "scaler_version=2"
+    - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v3-1562267374-ab36457a07f081fe-------z}"
   retry:
     automatic:
       - <<: *agent_transients
   timeout_in_minutes: 60
+  plugins:
```

Comment:
Do we need to use quad machines, and also are these plugins necessary? I don't have context on this.

Reply:
These settings were copied in from the queue in use by the same pipeline on the [`release`](https://github.com/spatialos/UnrealGDK/blob/release/.buildkite/premerge.steps.yaml) and [`master`](https://github.com/spatialos/UnrealGDK/blob/master/.buildkite/premerge.steps.yaml) branches.  This long-lived branch was just missed when those were updated.